src/Servers/Kestrel/Core/src/Features/IHttpWebTransportSessionFeature.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Versioning;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Features;
+
+namespace Microsoft.AspNetCore.Http.Features;
+
+/// <summary>
+/// API for accepting and retrieving WebTransport sessions.
+/// </summary>
+[RequiresPreviewFeatures("WebTransport is a preview feature")]
+public interface IHttpWebTransportSessionFeature
```

Comment:
Add a property like `IsWebTransportRequest`, similar to WebSockets. That way people know if it's safe to call Accept.
https://github.com/dotnet/aspnetcore/blob/7ae0d9f1f8e4ccf333bdf0d0baf17b3235256e71/src/Http/Http.Features/src/IHttpWebSocketFeature.cs#L16

Reply:
Technically I had this sort of logic implemented implicitly via the feature being null or not. However, I agree that this property is clearer than the null check method.